Cloud Application Manager/Administering Your Organization/resource-tags.md

```diff
@@ -2,11 +2,28 @@
 "title": "Tag Instances",
 "date": "10-30-2017",
 "author": "",
+"keywords": ["cam", "tags", "settings", "organization tag", "organization tag", "reporting"],
 "attachments": [],
 "contentIsHTML": false
 }}}
 
 ### Tag Instances
+**In this article:**
+
+* [Overview](#overview)
+* [Audience](#audience)
+* [Prerequisites](#prerequisites)
+* [Preset or Custom Tags](#preset-or-custom-tags)
+* [Applying Tags for Your Organization](#applying-tags-for-your-organization)
+* [Reporting on Cloud Application Manager Tags](#reporting-on-cloud-application-manager-tags)
+* [Reporting on AWS](#aws)
+* [Reporting Google Cloud](#google-cloud)
```

Comment:
```suggestion
* [Reporting on Google Cloud](#google-cloud)
```
And please, do not only change the title in the TOC, but also in the corresponding section title. Do it for everyone of these.